src/test/fuzz/util.h

```diff
@@ -33,6 +33,26 @@
 #include <vector>
 
 NODISCARD inline std::vector<uint8_t> ConsumeRandomLengthByteVector(FuzzedDataProvider& fuzzed_data_provider, const size_t max_length = 4096) noexcept
+{
+    std::vector<uint8_t> result;
```

Comment:
But if a suppression is added for `*/include/c++/*/memory.h` then other legitimate complaints will also be suppressed. As a temporary measure it works, but I think eventually it would be ideal to have the least amount of suppressions possible?

Reply:
Suppression added.